.jenkins/pytorch/macos-test.sh

```diff
@@ -15,6 +15,7 @@ if [ ! -d "${PYTORCH_ENV_DIR}/miniconda3" ]; then
 fi
 export PATH="${PYTORCH_ENV_DIR}/miniconda3/bin:$PATH"
 source ${PYTORCH_ENV_DIR}/miniconda3/bin/activate
+conda install python=3.6
```

Comment:
Why don't we run the build and test all in one go? There's no GPU consideration here. Would save a lot of time reinstalling stuff again second time around too.

Reply:
I am thinking about implementing the parallel testing solution soon for OSX tests (along with other Linux tests that take a long time). Separating them would be the first step on that path, although we can do them in one go for now and change it later.